`azurerm_key_vault_key` - `rotation_policy` block added (#19113)
hashicorp/terraform-provider-azurerm: website/docs/r/key_vault_key.html.markdown

* `tags` - (Optional) A mapping of tags to assign to the resource.

* `rotation_policy` - (Optional) A `rotation_policy` block as defined below.

---

A `rotation_policy` block supports the following:

> * `expiry_time` - (Optional) Expire a Key Vault Key after given time as an [ISO 8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations).

Reviewer: ```suggestion
* `expiry_time` - (Optional) Expire a Key Vault Key after given duration as an [ISO 8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations).
```

Author: Done

* `auto_rotation` - (Optional) An `auto_rotation` block as defined below.

> * `notification_time` - (Optional) Notify at a given time before expiry as an [ISO 8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations).

Reviewer: ```suggestion
* `notification_time` - (Optional) Notify at a given duration before expiry as an [ISO 8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations).
```

Author: Done

---

> An `auto_rotation` block supports the following:
> 
> * `time_after_create` - (Optional) Rotate automatically at a given time after create as an [ISO 8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations).
> 
> * `time_before_expiry` - (Optional) Rotate automatically at a given time before expiry as an [ISO 8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations).

Reviewer: this might be better as
```suggestion
An `automatic` block supports the following:

* `time_after_creat` - (Optional) Rotate automatically at a given time after create as an [ISO 8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations).

* `time_before_expiry` - (Optional) Rotate automatically at a given time before expiry as an [ISO 8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations).
```

so in tf it is `rotation_policy.automatic.time_after_creation` vs  `rotation_policy.auto_rotate.time_after_creation`

Author: Implemented as suggested
